ci/buildspec/build_public.yml

```diff
@@ -47,6 +47,7 @@ phases:
         --build-arg "API_PASS_KEY=${API_PASS_KEY}"
         --build-arg "GOOGLE_MAPS_API_KEY=${GOOGLE_MAPS_API_KEY}"
         --build-arg "GOOGLE_MAPS_MAP_ID=${GOOGLE_MAPS_MAP_ID}"
+        --build-arg "SHOW_ALL_MAP_PINS=${SHOW_ALL_MAP_PINS}"
```

Comment:
note: lets add this variable to `true` on dev and `false` on stage and prod before we merge this PR

Reply:
Updated!

Reply:
I did it in the same way we do backend vars but for the public site

Reply:
In that case you shouldn't have to have the values in this file. These are for build time variables rather than runtime ones

Reply:
Does that mean that the two above shouldn't be here either? Was following the pattern of the other runtime vars

Reply:
I don't think so. But I do see that those values are in the code build as environment variables. If we want to be extra safe it doesn't hurt to have them here for now and we can verify if they aren't in the AWS code build environment variable list doe the map pin functionality get picked up. Then we can clean these up later

Reply:
Also if that doesn't make sense we can discuss outside of github as well

Reply:
That sounds great!